### PR TITLE
Filter regions using blacklist or whitelist

### PR DESCRIPTION
**Pull request by collinped**
## Feature Description

Ability to filter regions using a whitelist or blacklist

## Solution Description

Allows the ability to filter regions returned by the country using a whitelist or blacklist property

### CODE REVIEW

**Comment by gehrj** [comment]
sorry for delay life has been busy, thanks for contributing i will look at this today.

**Review by gehrj — COMMENTED** [review]
why are we changing the repository?

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 .DS_Store
 node_modules
-/dist
```
Comment (gehrj) [inline .gitignore:3]: why remove this from gitignore?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "vue-country-region-select",
   "version": "2.0.13",
   "description": "A Country select component and Region select component for Vue.js. Work together and standalone.",
-  "repository": "https://github.com/gehrj/vue-country-region-select",
+  "repository": "https://github.com/collinped/vue-country-region-select",
```
Comment (gehrj) [inline package.json]: ?

**Comment by gehrj** [comment]
@collinped the changes to the white list look good, i have it pulled down and fixed the merge conflicts but I do not understand the other changes you are making, Will need more explanation or a revision with those removed to merge this.

**Comment by collinped** [comment]
@gehrj Sorry about that I ended up having to push a repository temporarily for a project in production. You can ignore the changes to the repository name and the linter changes. Thanks for the awesome package and for implementing that whitelist/blacklist into regions as well!

**Comment by gehrj** [comment]
@collinped sorry for delay things got kinda crazy for a minute... should get this in today

**Comment by gehrj** [comment]
this is now merged i had to use previous commit and push through that thanks again

**Pull request closed without merging**